packages/client/wallets/aa/src/utils/auth.ts

```
@@ -0,0 +1,10 @@
+export const parseToken = (token: any) => {
+    try {
+        const base64Url = token.split(".")[1];
+        const base64 = base64Url.replace("-", "+").replace("_", "/");
+        return JSON.parse(window.atob(base64 || ""));
+    } catch (err) {
+        console.error(err);
+        return null;
```

let's throw an Exception here